includes/Data/Patterns.php

```diff
@@ -0,0 +1,194 @@
+		return array(
+			'yith-wonder' => array(
+				'theme-styles'    => array(
+					'site-header-left-logo-navigation-inline' => array(
+						'active' => true,
+					),
+					'homepage-1'  => array(
+						'active'  => true,
+						'shown'   => true,
+						'combine' => true,
+					),
+					'site-footer' => array(
+						'active' => true,
+					),
+				),
+				'homepage-styles' => array(
+					'site-header-left-logo-navigation-inline' => array(
+						'active' => true,
+					),
+					'homepage-1'  => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'homepage-2'  => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'homepage-3'  => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'site-footer' => array(
+						'active' => true,
+					),
+				),
+				'site-pages'      => array(
+					'company-page'      => array(
+						'active'      => true,
+						'title'       => 'About',
+						'selected'    => true,
+						'shown'       => true,
+						'description' => __( 'Explain your company values or the history behind your brand.', 'wp-module-onboarding' ),
+					),
+					'contact-us'        => array(
+						'active'      => true,
+						'selected'    => true,
+						'title'       => 'Contact',
+						'shown'       => true,
+						'description' => __( 'Offer visitors a single page with a contact form, your street address and social media.', 'wp-module-onboarding' ),
+					),
+					'testimonials-page' => array(
+						'active'      => true,
+						'title'       => 'Testimonials',
+						'selected'    => false,
+						'shown'       => true,
+						'description' => __( 'Highlight your success with testimonials from your fans.', 'wp-module-onboarding' ),
+					),
+					'blog-page'         => array(
+						'active'      => true,
+						'selected'    => true,
+						'title'       => 'Blog',
+						'shown'       => true,
+						'description' => __( 'A page for periodic news, announcements and ideas.', 'wp-module-onboarding' ),
+					),
+				),
+				'header-menu'     => array(
+					'site-header-left-logo-navigation-inline' => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'homepage-1'                => array(
+						'active' => true,
+					),
+					'site-footer'               => array(
+						'active' => true,
+					),
+					'site-header-left-logo-navigation-below' => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'site-header-centered'      => array(
+						'active' => true,
+						'shown'  => true,
+					),
+					'site-header-splitted-menu' => array(
+						'active' => true,
+						'shown'  => true,
+					),
+				),
+				'site-features'     => SiteFeatures::get_site_features()[Data::current_flow()]
+			),
+		);
+	}
+
+	public static function cleanup_wp_grammar( $content ) {
+
+		// Remove template-part if that exists
+		$content = preg_replace( '/^<!-- wp:template-part .* \/-->$/m', '', $content );
```

Comment:
So basically in a Live Preview the template parts are not rendered as they have sub dependencies (as they require blocks themselves), so on the base level the grammar should be free of them, and this function just removes such occurrences so that they don't appear with an error for us in front End Live Preview.

<img width="903" alt="Screenshot 2022-12-13 at 10 07 54 AM" src="https://user-images.githubusercontent.com/48691514/207227948-b3cc3bbf-779a-476b-9806-9f9711b1f99c.png">
This is how template parts render out so we basically remove them in LivePreviews

Reply:
To add some reference to this, certain themes like Wonder use template parts inside their patterns which does not align with https://wordpress.org/support/article/comparing-patterns-template-parts-and-reusable-blocks/#guidelines, this is a check from our side to ensure that any theme that has a template-part inside a pattern does not break the preview